apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/DatasetsResourceTest.java

```diff
@@ -4560,25 +4573,28 @@ private ColumnType getType(Map.Entry<String, JsonNode> entry) {
         };
     }
 
-    private void assertDatasetItemExperiments(DatasetItemPage actualPage, List<DatasetItem> items,
+    private void assertDatasetItemExperiments(DatasetItemPage actualPage, List<DatasetItem> datasetItems,
             List<ExperimentItem> experimentItems) {
 
-        assertPage(List.of(items.getFirst()), List.of(actualPage.content().getFirst()));
+        var actualDatasetItems = actualPage.content();
+        assertPage(datasetItems, actualDatasetItems);
 
-        var actualExperimentItems = actualPage.content().getFirst().experimentItems();
-        assertThat(actualExperimentItems).hasSize(1);
-        assertThat(actualExperimentItems.getFirst())
-                .usingRecursiveComparison()
-                .ignoringFields(IGNORED_FIELDS_LIST)
-                .isEqualTo(experimentItems.getFirst());
+        for (int i = 0; i < actualDatasetItems.size(); i++) {
```

Comment:
Minor: can you assert on the expected size of `actualDatasetItems`?

Reply:
Updated